examples/np/cem_cartpole.py

```diff
@@ -35,7 +35,9 @@ def cem_cartpole(ctxt=None, seed=1):
         policy = CategoricalMLPPolicy(name='policy',
                                       env_spec=env.spec,
                                       hidden_sizes=(32, 32))
-
+        input_var = env.observation_space.to_tf_placeholder(name='obs',
```

Comment:
Is there a strong reason this can't happen in the policy? It makes the experiment launcher a lot less obvious to me.

Reply:
Indeed, we can. It's just it no longer is a `np` algorithm (which it actually isn't in the first place, given it's using tf primitives)

Reply:
or we can use a `np` primitive?

Reply:
I think we will have to consider this relationship later as we write the primitive interfaces, so for now just do whatever is easiest. This algorithm is most interesting to KR and I, and fewer users actually use it, anyway.